tools/online_editor/index.html

```diff
@@ -1,58 +1,73 @@
-        integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+  <head>
+    <meta charset="utf-8" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+      integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
+      crossorigin="anonymous"
+    />
     <title>Slint Online Editor</title>
     <style>
-        canvas {
-            touch-action: none;
-        }
+      canvas {
+        touch-action: none;
+      }
     </style>
     <script type="module" src="./index.ts"></script>
-</head>
+  </head>
 
-<body>
-    <p>This is a prototype for an online editor for the Slint language. More info <a
-            href="https://github.com/slint-ui/slint">github.com/slint-ui/slint</a> </p>
-    <p>Select a demo to load: <select id="select_combo">
-            <option value="">-- Select a demo --</option>
-            <option value="examples/gallery/gallery.slint">Gallery</option>
-            <option value="examples/printerdemo/ui/printerdemo.slint">Printer demo</option>
-            <option value="examples/todo/ui/todo.slint">Todo demo</option>
-            <option value="examples/iot-dashboard/main.slint">IOT Dashboard</option>
-        </select></p>
-    <p><button type="button" id="compile_button">Compile!</button>
-        &nbsp; &nbsp;
-        <input type="checkbox" id="auto_compile" name="auto_compile" checked>
-        <label for="auto_compile">Automatically compile on change</label>
-        <!--
+  <body>
+    <p>
+      This is a prototype for an online editor for the Slint language. More info
+      <a href="https://github.com/slint-ui/slint">github.com/slint-ui/slint</a>
+    </p>
+    <p>
+      Select a demo to load:
+      <select id="select_combo">
+        <option value="">-- Select a demo --</option>
+        <option value="examples/gallery/gallery.slint">Gallery</option>
+        <option value="examples/printerdemo/ui/printerdemo.slint">
+          Printer demo
+        </option>
+        <option value="examples/todo/ui/todo.slint">Todo demo</option>
+        <option value="examples/iot-dashboard/main.slint">IOT Dashboard</option>
+      </select>
+    </p>
+    <p>
+      <button type="button" id="compile_button">Compile!</button>
+      &nbsp; &nbsp;
+      <input type="checkbox" id="auto_compile" name="auto_compile" checked />
+      <label for="auto_compile">Automatically compile on change</label>
+      <!--
         &nbsp; &nbsp; &nbsp; &nbsp;
         <label for="style_combo">Style:</label>
         <select id="style_combo">
             <option value="fluent">fluent</option>
         </select>
         -->
-        &nbsp; &nbsp; &nbsp; &nbsp;
-        <a href="../docs/rust/slint/docs/langref/index.html">Slint Language Reference</a>
+      &nbsp; &nbsp; &nbsp; &nbsp;
+      <a href="../docs/rust/slint/docs/langref/index.html"
+        >Slint Language Reference</a
```

Comment:
(i'm not a fan of this style that cut the tags like this, in the middle)

(also, if the prettier is different that the one that simon and i use, then that means it may be reformatted again with our formater everytime we make edits)

Reply:
I do not care too much -- as long as I can automate the issue:-)